fiware-region-sanity-tests/.noserc

```diff
@@ -0,0 +1,4 @@
+[nosetests]
+logging-filter=TestCase,novaclient,neutronclient
+logging-level=INFO
+logging-format=%(asctime)s UTC/GMT - %(name)s - %(levelname)s - %(message)s
```

Comment:
Please remove "GMT", as UTC supersedes that timezone (no matter conversion function is called `time.gmtime`).